src/NuGet.Clients/VisualStudio/Extensibility/IVsPackageInstaller.cs

```diff
@@ -18,10 +18,19 @@ public interface IVsPackageInstaller
         /// <summary>
         /// Installs a single package from the specified package source.
         /// </summary>
-        /// <param name="source">The package source to install the package from.</param>
+        /// <param name="source">
+        /// The package source to install the package from. This value can be
+        /// <c>"All"</c> or <c>null</c> to indicate that the user's configured
```

Comment:
Remove all, this should only suggest null as the official way to get all sources.